src/plugins/intel_cpu/src/emitters/snippets/brgemm_base.hpp

```diff
@@ -0,0 +1,76 @@
+// Copyright (C) 2020-2025 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#pragma once
+
+#include "emitters/snippets/cpu_kernel_executor_table.hpp"
+#include "emitters/utils.hpp"
+#include "openvino/core/type/element_type.hpp"
```

Comment:
Looks like unused. Can we remove it?

Reply:
`OV_CPU_JIT_EMITTER_ASSERT `#include "emitters/utils.hpp"`, 
`struct BrgemmGenericKernelConfig : public snippets::KernelExecutorBase::GenericConfig` need `#include "emitters/snippets/cpu_kernel_executor_table.hpp"`
#include "openvino/core/type/element_type.hpp" is unused and removed.